### ISSUE TITLE
Will this mod work on call of duty modern warfare 3 (2023)? Will it be detected by anti-cheat?

### ISSUE THREAD

**Issue by somebodylive** [Nukem9/dlssg-to-fsr3#142]
My in-game framerate isn't very high, and I'd like to increase it a bit with this mod

**Comment by Z4ndyz** [comment]
Yes it'll be detected hopefully wont even let you run if not even worst case you'll eventually get banned DO NOT USE IT IN MULTIPLAYER GAMES that have anticheat or official servers, use it in games like Valheim or Sons of Forest (ofc they dont have frame gen available at all but that's an example of multiplayer games where it's ok) but i repeat do not use it in any esport titles, competitive, mobas, mmos or any games with servers and anticheats or ToS against game file modification or injection do not use it in modern warfare 3 

offtopic there was a rumour that fsr 3.0 is mentioned in ingame files it might be added officialy

**Comment by Dravic** [comment]
@Z4ndyz good tips.

I would also caution against using it in Diablo IV.
